packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -295,6 +299,26 @@ export class DefaultWeb3Handler implements Web3Handler, FullnodeHandler {
     return block
   }
 
+  public async getBlockByHash(
+    blockHash: string,
+    fullObjects: boolean
+  ): Promise<any> {
+    log.debug(`Got request to get block ${blockHash}.`)
+    const res: object = await this.context.provider.send(
+      Web3RpcMethods.getBlockByHash,
+      [blockHash, fullObjects]
+    )
+    const block = this.parseInternalBlock(res, fullObjects)
+
```

Comment:
need to set timestamp as well

Reply:
So I think that you shouldn't need to explicitly set the timestamp because I use `this.parseInternalBlock(...)` which should do this for us.